src/MetaCompilation.Analyzers/Core/xlf/MetaCompilationResources.cs.xlf

```diff
@@ -5,15 +5,15 @@
       <trans-unit id="AnalyzerDescription">
         <source>Type names should be all uppercase.</source>
         <target state="new">Type names should be all uppercase.</target>
-        <note>An optional longer localizable description of the diagnostic.</note>An optional longer localizable description of the diagnostic.</trans-unit>
+        <note />An optional longer localizable description of the diagnostic.</trans-unit>
```

Comment:
Hmm, I can't repro this. I just copied the resx file into a temp project using xliff v53 and I'm getting the right result:

![image](https://user-images.githubusercontent.com/75470/28354049-2dc4dc78-6c13-11e7-86aa-c146da3f308f.png)

Try cleaning intermediates, deleting all xlf files, and regenerating again.